animation_nodes/data_structures/sounds/sound.py

```diff
@@ -0,0 +1,44 @@
+import numpy
+from functools import lru_cache
+from math import ceil, log, isclose
+
+class Sound:
+    def __init__(self, soundSequences):
+        self.soundSequences = soundSequences
+
+    def getSamplesInRange(self, start, end):
+        if end <= start: raise ValueError("Invaild range!")
+        maxSampleRate = max(sequence.data.sampleRate for sequence in self.soundSequences)
```

Comment:
Somehow it feels wrong that you evaluate some sounds using a wrong sample rate, or do I not read that correctly?

Reply:
You are right, this should be a constant sample rate.